src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const employsList = [...document.querySelectorAll('li')];
+const listParent = document.querySelector('ul');
+
+function sortList(list) {
+  list.sort((a, b) => {
```

Comment:
Sort is a mutable method, so it's better to create a copy of list inside the function to be sure that it doesn't mutate array from parameter